lua/ts_context_commentstring/internal.lua

```diff
@@ -129,13 +130,13 @@ function M.check_node(node, language_config, commentstring_key)
   return M.check_node(node:parent(), language_config, commentstring_key)
 end
 
----Attach the module to the current buffer
+---@deprecated
 function M.attach()
+  vim.deprecate("context_commentstring module", "use require('ts_context_commentstring').setup {} and set vim.g.skip_ts_context_commentstring_module = true to speed up loading", "in the future", "ts_context_commentstring")
```

Comment:
This is how the deprecation message looks like right now:

```
   Warn  20:49:15 notify.warn context_commentstring module is deprecated, use use require('ts_context_commentstring').setup {} and set vim.g.skip_ts_context_commentstring_module = true to speed up loading instead.
This feature will be removed in ts_context_commentstring version in the future
```

I'd make two small changes here:

```suggestion
  vim.deprecate("context_commentstring Treesitter module", "require('ts_context_commentstring').setup {} and set vim.g.skip_ts_context_commentstring_module = true to speed up loading", "in the future", "ts_context_commentstring")
```

just to make the text look a bit better. Resulting in:

```
   Warn  20:52:29 notify.warn context_commentstring Treesitter module is deprecated, use require('ts_context_commentstring').setup {} and set vim.g.skip_ts_context_commentstring_module = true to speed up loading instead.
This feature will be removed in ts_context_commentstring version in the future
```

What do you think?

Reply:
I'll specify "nvim-treesitter", as "Treesitter" could potentially be mistaken for the parser framework itself.